besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1912,6 +1917,18 @@ public void validateRpcOptionsParams() {
     }
   }
 
+  public void validateChainDataPruningParams() {
```

Comment:
Should we also check that they're not trying to use it with FOREST mode? (or only allow it for BONSAI)

Reply:
Technically it is fine to use with FOREST